lib/okta/profile/configuration.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+require 'okta/configuration.rb'
+
+module Okta
+  module Profile
+    class Configuration < Okta::Configuration
+      def base_path
+        Settings.oidc.profile_api_url || 'https://deptva-eval.okta.com/api/v1/users/'
```

Comment:
If `base_path` is the only thing leading you to have separate OktaService and OktaProfileService, then it's fair game to have base_path be `https://deptva-eval.okta.com/api/v1` and then have a `get_profile` method that sets `url` to `users/ + uid` (or do so with path.join or whatever).

Reply:
And I would not default the base_path; I wouldn't want us to inadvertently make requests to the eval org in production. Practically the risk is very low, we wouldn't have the right token, etc, but I don't think we typically do this elsewhere. If it was done to make specs work, then there is a way to factor out the actual URL and other values from VCR cassettes - so you run them the first time with the real value in settings.local.yml, but it puts in a placeholder value into the recorded request/response.

Reply:
I'll refactor the base_path requirement in the config to not be set or dynamically changeable before I do anything with VCR or start recording responses.